pwndbg/aglib/heap/ptmalloc.py

```diff
@@ -56,6 +56,7 @@
 # Only import it globally during static type checking.
 if typing.TYPE_CHECKING:
     import pwndbg.aglib.heap.structs
+    import pwndbg.aglib.heap.structs.TCACHE_FILL_COUNT
```

Comment:
```suggestion
```
this import is not needed

Reply:
anyway, you don't need `force push`, it is easier for us to review new commits, without rebasing/force pushing